### DISCUSSION TITLE
Workflow Notification for incoming emails

### DISCUSSION THREAD

**Discussion by willinmon** [YetiForceCompany/YetiForceCRM#14423] [How to]
Is there a way to trigger a notification when a new email is appended to a ticket? I am not seeing the option in the workflow creator. 

Our team is using emails to communicate with customers and they send/receive emails through the ticket and not the mailbox. When a new email comes in it is automatically related to the correct ticket but I need a way to notify the team member that they have a new message in their ticket. With possibly hundreds of tickets coming in a day it would be impossible to keep up with them without some kind of notification. Is there a way to setup a workflow to do this?

**Comment by mariuszkrzaczkowski** [comment]
Email notifications in the mail scanner have been improved in the latest version. It's possible to do that in workflow as well as in a custom handler.
https://gitdeveloper.yetiforce.com/index.php?module=Workflows&parent=Settings&page=1&view=List&sourceModule=HelpDesk
